src/Types/Output/InterfaceType.hack

```diff
@@ -4,9 +4,15 @@ use namespace HH\Lib\{Dict, Vec};
 use namespace Slack\GraphQL;
 
 abstract class InterfaceType extends CompositeType {
+    abstract const keyset<classname<GraphQL\Types\ObjectType>> POSSIBLE_TYPES;
 
     <<__Override>>
     final public function getKind(): GraphQL\Introspection\__TypeKind {
         return GraphQL\Introspection\__TypeKind::INTERFACE;
     }
+
+    <<__Override>>
+    final public function getPossibleTypes(): vec<GraphQL\Introspection\__Type> {
+        return Vec\map(static::POSSIBLE_TYPES, $type ==> $type::nonNullable()->nullableForIntrospection());
```

Comment:
This wasn't super intuitive. At first I did `nonNullable` and that doesn't work because then we return these with the non null wrapper. `$type::nullableOutput` works but I think the intent is that we return `nullableForIntrospection`.

Reply:
It doesn't really matter, but I think `nullableOutput()` is probably preferable (less confusing).

I added `nullableForIntrospection()` only so that we had something to call from places that don't know if a given type is an input/output type, so they don't know whether to call `nulllableOutput()` or `nullableInput()`. There should be no need to call `nullableForIntrospection()` from anywhere except the few introspection-specific places  that currently do.